.github/maintainers_guide.md

```diff
@@ -24,23 +24,33 @@ Run this from the root of the project
 
 ### Testing
 
-Test can be run directly with Deno.
+Test can be run directly with Deno:
 
-```
-deno test src
-```
+    deno task test
+
+You can also run a test coverage report with:
 
+    deno task coverage
 
 ### Releasing
 
-* Releases for this library are triggered off of tags. You should create a new GitHub Release, and use a new version that adheres to semantic versioning based on what's being release. Version tags should match the following pattern: `1.0.1` (no `v` preceding the number).
-* Once the release has been created, a corresponding version will be avaliable on https://deno.land/x/deno_slack_api.
+Releases for this library are automatically generated off of git tags. To create a new release:
+
+1. Create a new GitHub Release from the [Releases page](https://github.com/slackapi/deno-slack-api/releases) by clicking the "Draft a new release" button.
+2. Input a new version manually into the "Choose a tag" input. Ensure that this version adheres to [semantic versioning][semver] based on what's being released. Version tags should match the following pattern: `1.0.1` (no `v` preceding the number).
+  - After you input the new version, click the "Create a new tag: x.x.x on publish" button.
+3. Set the "Target" input to the "main" branch.
+4. Name the release title after the version tag.
+5. You can copy the [release contents of the 0.0.3 release](https://github.com/slackapi/deno-slack-api/releases/edit/0.0.3) to use as a template for the release notes. Make sure to call out the changes under "What's Changed" and shout out any new contributors under "New Contributors". Also don't forget to update the detailed changelog link at the bottom to reference the latest and previous release correctly!
```

Comment:
We've actually just been using the "Auto-generate release notes" button that does this for us, I think we can recommend that instead. Maybe call out to review the notes and make sure they're more reader friendly when applicable.